code/modules/vore/eating/belly_vr.dm

```diff
@@ -143,10 +148,17 @@
 
 	prey.forceMove(owner)
 	internal_contents.Add(prey)
+
+//	var/datum/belly/B = check_belly(owner)
+//	if(B.silenced == FALSE) //this needs more testing later
 	prey << sound('sound/vore/prey/loop.ogg', repeat = 1, wait = 0, volume = 35, channel = CHANNEL_PREYLOOP)
 
+	owner.updateVRPanel()
+	for(var/mob/living/M in internal_contents)
+		M.updateVRPanel()
+
 	if(inside_flavor)
-		prey << "<span class='notice'><B>[inside_flavor]</B></span>"
+		to_chat(prey, "<span class='notice'>[inside_flavor]</span>")
```

Comment:
I still don't know why this isn't working, btw. It's on the to-fix list eventually.